homework/index.js

```diff
@@ -30,18 +30,100 @@
     return elem;
   }
 
+  function createInfoDiv(repoDiv, selectRepo) {
+    const tableDiv = createAndAppend('div', repoDiv, { class: 'left-div' });
+    const infoTable = createAndAppend('table', tableDiv, { class: 'contributor-table' });
+    const tableBody = createAndAppend('tbody', infoTable);
+    const rowName = createAndAppend('tr', tableBody);
+    const repoName = createAndAppend('td', rowName, { text: 'Repository:', class: 'repo-name' });
+    createAndAppend('a', repoName, {
+      target: '_blank',
+      href: selectRepo.html_url,
+      text: selectRepo.name,
+    });
+    const rowDesc = createAndAppend('tr', tableBody);
+    createAndAppend('td', rowDesc, {
+      text: `Description: ${selectRepo.description}`,
+      class: 'repo-desc',
```

Comment:
Do not assign unused classes. You have no associated styling for this class. The same applied for ids: if you don't need an id for styling or for use with `getElementById` don't assign one. The same applies to other places in your code.